users/utils.py

```diff
@@ -37,3 +42,53 @@ def user_has_change_permission(user, *models):
 
 def user_has_delete_permission(user, *models):
     return _user_has_models_perms("delete", models)(user)
+
+
+def is_customer(user):
+    if user.is_staff or user.is_superuser:
+        return False
+    elif (
+        user.groups.count() == 1
+        and user.groups.first().name == settings.CUSTOMER_GROUP_NAME
+    ):
+        return True
+    else:
+        return False
+
+
+_berth_customers_group = None
+
+
+def get_berth_customers_group():
+    global _berth_customers_group
+    if not _berth_customers_group:
+        _berth_customers_group = Group.objects.get(name=settings.CUSTOMER_GROUP_NAME)
+    return _berth_customers_group
```

Comment:
Can we use the power of Python for this one? E.g. [lru_cache](https://docs.python.org/dev/library/functools.html#functools.lru_cache):

```suggestion
@lru_cache(maxsize=32)
def get_berth_customers_group():
    return Group.objects.get(name=settings.CUSTOMER_GROUP_NAME)
```
(`maxsize` is an arbitrary number, but since we don't expect the value to change often, we can keep it as a small number)